svn/common.py

```diff
@@ -49,10 +51,13 @@ def run_command(self, subcommand, args, success_code=0,
 
         _logger.debug("RUN: %s" % (cmd,))
 
+        environment_variables = copy.deepcopy(os.environ)
```

Comment:
It's not valid to provide anything but a string as a value in this dictionary. It will cause an error. Even if it were possible, it wouldn't be Pythonic (read: intuitive and useful) as anything that wasn't a string would be *coerced* to a string.